packages/components/app/styles/components/alert.scss

```diff
@@ -0,0 +1,263 @@
+  }
+
+  // Notice: in the future this may become a "Link::Inline" component (for now we declare the styles directly here)
+  a {
+    color: var(--token-color-foreground-action);
+    // At the moment the "focus" state is not well defined in design (the one that is in Figma does not work) so we just apply a simple color to the default outline
+    outline-color: var(--token-color-focus-action-external);
+
+    &:hover {
+      color: var(--token-color-foreground-action-hover);
+    }
+
+    &:active {
+      color: var(--token-color-foreground-action-active);
+    }
+  }
+}
+
+// ACTIONS
+
+.hds-alert__actions {
+  align-items: center;
+  display: flex;
+
+  > * {
+    margin-top: 16px;
+  }
+
+  > * + * {
+    margin-left: 16px;
+  }
+}
+
+// DISMISS
+
+.hds-alert__dismiss {
+  background-color: transparent;
+  border: none;
+  color: var(--token-color-foreground-faint);
+  cursor: pointer;
+  flex: none;
+  margin-left: 16px;
+  margin-top: 2px; // for alignment with the main icon and text
+  padding: 0;
+
+  .hds-alert--type-compact & {
+    margin-top: 1px;
+  }
+
+  &:hover,
+  &.is-hover {
+    &::before { // we re-use the pseudo-element created by the "focus-ring" mixin
+      background-color: rgba(#dedfe3, 0.4);
+    }
+  }
+
+  // notice: this is used not only for the focus, but also to increase the clickable area
+  @include hds-focus-ring-with-pseudo-element($top: -4px, $right: -4px, $bottom: -4px, $left: -4px);
+
+  &:active,
+  &.is-active {
+    color: var(--token-color-foreground-secondary);
+    &::before {
+      background-color: rgba(#dedfe3, 0.4);
+      border: 1px solid var(--token-color-border-strong);
+    }
+  }
+}
+
+
+// TYPES
+
+.hds-alert--type-page {
+  padding: 16px 48px; // by design
+}
+
+.hds-alert--type-inline {
+  border-radius: 6px;
+  border-style: solid;
+  border-width: 1px;
+  padding: 16px;
+}
+
+.hds-alert--type-compact {
+  .hds-alert__icon {
+    margin-right: 8px;
+    margin-top: 2px; // notice: the icon size for the "compact" type is 14px, not 20px, so this 2px extra are needed to reach the exact same height as the "description" text line-height
+  }
+
+  // extra safety
+  .hds-alert__title {
+    display: none;
+    & + .hds-alert__description {
+      margin-top: 0;
+    }
+  }
+}
+
+// COLORS (& TYPES)
+
+.hds-alert--color-neutral {
+
+  &.hds-alert--type-page {
+    background-color: var(--token-color-surface-faint);
+    box-shadow: 0px 1px 0px 0px var(--token-color-palette-alpha-300);
+  }
+
+  &.hds-alert--type-inline {
+    background-color: var(--token-color-surface-faint);
+    border-color: var(--token-color-border-strong); // notice: in the "neutral" color the "inline" has a slightly darker border color compared to the others to increase contrast (eg. could be used on a light gray background)
+  }
+
+  &.hds-alert--type-compact {
+    // special override by design
+    .hds-alert__icon {
+      color: var(--token-color-foreground-faint);
```

Comment:
This override applies to `page`, `inline` type alerts and `toast` as well.

Reply:
fixed in https://github.com/hashicorp/design-system/pull/245/commits/21e86e010aa5f65d0f332ffc2288f9b3071d8511